src/pages/index.js

```diff
@@ -95,6 +95,11 @@ var Index = React.createClass({
           }
         </head>
         <body>
+          <div className="share-progress-holder">
+            <div className="sp_182920 sp_fb_small"></div>
```

Comment:
I assume the `sp_182920` etc are dictated by share progress, does it make sense to do something like
```
const ShareProgressClasses = {
  facebook: "sp_182920 sp_fb_small",
  twitter: "sp_... sp_...",
  ...
};

...

render() {
  return ...
    <div className=`${ShareProgressClasses.facebook}` />
    ...
}
```